WcaOnRails/app/models/competition.rb

```diff
@@ -824,6 +824,25 @@ def to_wcif
     }
   end
 
+  def load_wcif!(wcif)
```

Comment:
The distinction I'm trying to make is between storing JSON text and populating Rails model tables.

I know we have unresolved philosophical differences there.